### DISCUSSION TITLE
fancyBox v3.5.7 + Vimeo autoplay

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [fancyapps/ui#95] [General]
Hi, 

I'm using fancyBox `v3.5.7` with both YouTube and Vimeo videos. The YouTube videos auto play in a modal with no additional configuration. The Vimeo videos won't autoplay unless I add `&amp;muted=1`. The video plays, but it's muted. 

Just wondering is there a workaround for this? 

Is there something happening with fancyBox + YouTube, that's not being done with Vimeo?

**Comment by fancyapps** [comment]
Hi,

In case you missed it, "Chrome's autoplay policies changed in April of 2018", see https://developer.chrome.com/blog/autoplay/

So, the answer is - unfortunately nothing can be done here.

**Reply by stephenmeehanuk** [reply]
Thanks, I spotted that while looking for answers to this. 

Just wondering how come YouTube embeds work with autoplay but Vimeo doesn't?

I'm displaying the video in a modal, that's activated on a mouse click. So the user is requesting the video to play.

**Reply by fancyapps** [reply]
Could you please try the latest version and see if it works better?
